src/main/java/org/cru/aemscraper/Main.java

```diff
@@ -123,14 +123,7 @@ private static void sendToS3(final String[] args, final String rootUrl) throws E
             LOG.info("Parsing pages ...");
             pageParsingService.parsePages(rootEntity, ALL_PAGE_DATA);
             LOG.info("Parsing pages ... done {}", rootEntity.getJcrContent());
-            Set<Template> desiredTemplates = ImmutableSet.of(
-                STATIC_ARTICLE,
-                ARTICLE_LONG_FORM,
-                DAILY_CONTENT,
-                DYNAMIC_ARTICLE,
-                SUMMER_MISSION,
-                INTERNATIONAL_INTERNSHIP
-            );
+            Set<Template> desiredTemplates = ImmutableSet.of(STATIC_ARTICLE, ARTICLE_LONG_FORM);
```

Comment:
I started a local branch for using properties files for things like this, but never finished it. Probably not worth the effort for now, but just wanted you to know.